### PR TITLE
Logging es searching

### PR DESCRIPTION
**Pull request by abeachcatalyze**
I added some information on making requests to elasticsearch directly without Kibana.

### CODE REVIEW

File: logging.md [hunk 1]
```diff
@@ -147,4 +147,50 @@ Once you have created a custom dashboard and saved it you can set it to be your
...
+
+Elasticsearch can be queried directly by making a request to `https://podhostname/__es`
+
+The index and type of the documents are specified in the path of the uri.
+
+The index name is in the format `logstash-YYYY.MM.DD`.  For example, `logstash-2015.11.09`
+
+The type is "syslog"
+
+To return all the records in Elasticsearch from 2015-11-09, the uril would be `https://podhostname/__es/logstash-2015.11.09/syslog/_search`
```
Comment (andrewawagner) [inline logging.md]: "uril" to "uri"

**Comment by andrewawagner** [comment]
How do they handle the authentication for the curl request?

**Comment by abeachcatalyze** [comment]
I added some steps for using the -u or a netrc file for curl to send login credentials for authentication to access Elasticsearch

File: logging.md [hunk 2]
```diff
@@ -147,4 +147,70 @@ Once you have created a custom dashboard and saved it you can set it to be your
...
+machine podhostname
+  login myusername
+  password mypassword
+```
+
+Example:
+
+~/.netrc:
+```
+machine pod09999.catlayzeapps.com
```
Comment (andrewawagner) [inline logging.md]: typo in catalyzeapps

**Pull request merged**